<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Auth</class>
 <widget class="QWidget" name="Auth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>密钥生成</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;登陆以实现密钥加密与存储&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>登陆</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>密钥生成</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="1">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>10</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;密钥长度&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLineEdit" name="lineEdit">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>20</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>生成</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>非对称密钥</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QSpinBox" name="spinBox">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="value">
             <number>16</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;对称密钥&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>生成</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>20</height>
             </size>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string>密钥的密码（默认为空）</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="9" column="3" colspan="2">
      <widget class="QTextBrowser" name="textBrowser">
       <property name="minimumSize">
        <size>
         <width>382</width>
         <height>71</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;华文楷体&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#55aa7f;&quot;&gt;密钥管理&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
